requirements.txt

```diff
@@ -1,3 +1,5 @@
 Flask==1.1.1
 flask-sqlalchemy==2.4.1
 psycopg2-binary==2.8.5
+pytest==5.2.0
+pytest-cov==2.9.0
```

Comment:
You may want to separate the testing libraries to a different requirements file, say requirements-test.txt, Inside the tests folder, since they're not used in the app itself.